cmd/root.go

```diff
@@ -0,0 +1,38 @@
+/*
+Copyright © 2014-2021 Nimble
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+// rootCmd represents the base command when called without any subcommands
+var rootCmd = &cobra.Command{
+	Use:  "gin-templates",
```

Comment:
Just like for [Elixir templates](https://github.com/nimblehq/elixir-templates), we should prefix it with `nimble`:

 ```suggestion
	Use:  "nimble-gin",
```

We could also omit templates to sound cooler 😎

Reply:
Fixed in 0264db2eb9e3b9029c11c0dff523e88a7968722e